waisntechtools/alerts/templates/alerts/fetch_sms.html

```diff
@@ -0,0 +1,21 @@
+
+<table>
+    <tr>
+      <td>
+        Date
+      </td>
+      <td>
+        From
+      </td>
+      <td>
+        Message
+      </td>
+    </tr>
+  {% for message in messages %}
+  <tr>
+    <td>{{ message.date_sent }}</td>
+    <td>{{ message.from_ }}</td>
```

Comment:
Is the underscore in `from_` correct?

Reply:
Yes. Twilio docs indicates to use "from_"